docs/concepts/composition.md

```diff
@@ -1,156 +1,96 @@
-corresponding `MySQLInstance` claim.
-
-> Note that composite resources and compositions are _cluster scoped_ - they
-> exist outside of any Kubernetes namespace. A claim is a namespaced proxy for a
-> composite resource. This enables Crossplane to model complex relationships
-> between XRs that may span namespace boundaries - for example MySQLInstances
-> spread across multiple namespaces can all share a VPC that exists above any
-> namespace.
-
-## Creating A New Kind of Composite Resource
-
-New kinds of composite resource are defined by a platform builder. There are
-two steps to this process:
-
-1. Define your composite resource, and optionally the claim it offers.
-1. Specify one or more possible ways your composite resource may be composed.
-
-### Define your Composite Resource
-
-Composite resources are defined by a `CompositeResourceDefinition`:
+# Composite Resources
+
+## Overview
+
+Crossplane Composite Resources are opinionated Kubernetes Custom Resources that
+are _composed_ of [Managed Resources][managed-resources]. We often call them XRs
+for short.
+
+![Diagram of claims, XRs, and Managed Resources][xrs-and-mrs]
+
+> If you're already familiar with Composite Resources and looking for a detailed
+> configuration reference or some tips, tricks, and troubleshooting information,
+> try the [Composition Reference][xr-ref].
+
+Composite Resources are designed to let you build your own platform with your
+own opinionated concepts and APIs without needing to write a Kubernetes
+controller from scratch. Instead, you define the schema of your XR and teach
+Crossplane which Managed Resources it should compose (i.e. create) when someone
+creates the XR you defined.
+
+
+Below is an example of a Composite Resource:
+
+```yaml
+apiVersion: database.example.org/v1alpha1
+kind: XPostgreSQLInstance
+metadata:
+  name: my-db
+spec:
+  parameters:
+    storageGB: 20
+  compositionRef:
+    name: production
+  writeConnectionSecretToRef:
+    namespace: crossplane-system
+    name: my-db-connection-details
+```
+
+You define your own XRs, so they can be of whatever API version and kind you
+like, and contain whatever spec and status fields you need.
+
+## How It Works
+
+The first step towards using Composite Resources is configuring Crossplane so
+that it knows what XRs you'd like to exist, and what to do when someone creates
+one of those XRs. This is done using a `CompositeResourceDefinition` resource
+and one or more `Composition` resources.
+
+Once you've configured Crossplane with the details of your new XR you can either
+create one directly, or use a _claim_. Typically only the folks responsible for
+configuring Crossplane (often a platform or SRE team) have permission to create
+XRs directly. Everyone else manages XRs via a lightweight proxy resource called
+a Composite Resource Claim (or claim for short). More on that later.
+
+![Diagram combining all Composition concepts][how-it-works]
```

Comment:
I would add the abbrevations XRD, XR and MR to the diagram.

Reply:
I'd prefer not to. I think using the full names does enough to explain what each concept is, and feel adding the abbreviations would clutter it visually.

Reply:
I showed this document [how-it-works] to some new Crossplane users, and they really liked it! However, as we discussed it we felt making it clear which components are defined by the Platform Team, and what is generated dynamically by Crossplane could help. 

Could we emphasize in the diagram that the CompositeResource is a new k8s object generated within crossplane from the XR + Composition?

Reply:
> Could we emphasize in the diagram that the CompositeResource is a new k8s object generated within crossplane from the XR + Composition?

I think there might be a terminology mixup - this doesn't seem like an accurate representation of what happens.

In an ideal world we want to communicate:

1. A **type of** Composite Resource exists because it was defined by a Composite Resource Definition.
2. Ditto for a type of Claim.
3. An **instance of** a Claim exists when an app team member creates it.
4. An **instance of** a Composite Resource exists either:
     a. Because it was created automatically by Crossplane in response to a Claim.
     b. Because a platform team member created it.
5. An instance of a Composite Resource uses a Composition to determine what Managed Resources to compose.
6. Instances of Managed Resources exist because a Composite Resource created them (per its Composition)

My hunch is that this is way too much to fit in a diagram, and that we should have a slightly 'low resolution' diagram that makes more sense when you read the surrounding context (i.e. documentation).

That said, I'm open to suggestions for light weight changes we could make here. Perhaps we could:

* Add a "_Creates_ Composite Resource" label to the Claim -> Composite Resource relationship.
* Add a "_Created_ by Composite Resource" subtitle to the Managed Resource cluster.
* Change the Composite Resource Definition subtitle to "_Defines_ Composite Resource and Claim Types".
* Change the Composition subtitle to "_Template_ for Managed Resources"